documentation/assemblies/security/assembly-security.adoc

```diff
@@ -10,9 +10,12 @@ Communication between Kafka brokers (interbroker communication), between ZooKeep
 Communication between Kafka clients and Kafka brokers is encrypted according to how the cluster is configured.
 For the Kafka and {ProductName} components, TLS certificates are also used for authentication.
 
-The Cluster Operator automatically sets up TLS certificates to enable encryption and authentication within your cluster.
+The Cluster Operator automatically sets up and renews TLS certificates to enable encryption and authentication within your cluster.
```

Comment:
Do we need to clarify that it renews only its own certificates? Not any supplied by the user?